tests/approaches/test_oracle_approach.py

```diff
@@ -862,6 +862,27 @@ def test_oracle_approach_stick_point():
         "num_test_tasks": 2,
         "stick_point_num_points_train": [1],
         "stick_point_num_points_test": [2],
+        "stick_point_disable_angles": False,
```

Comment:
looks like a lot of duplicated code in this function... let's use the pytest parametrize pattern here? see `tests/approaches/test_gnn_metacontroller_approach.py` for an example of how to use it with multiple arguments

Reply:
you're right, but this entire file is a disaster in this respect. i'm going to make an issue to refactor the whole file.